hedera-mirror-common/src/test/resources/init.sql

```diff
@@ -31,6 +31,7 @@ revoke create on schema temporary from public;
 -- Add extensions
 create extension if not exists btree_gist;
 create extension if not exists pg_stat_statements;
+CREATE extension if not exists pg_trgm;
```

Comment:
```suggestion
create extension if not exists pg_trgm;
```

also add it to https://github.com/hashgraph/hedera-mirror-node/blob/ab647eabbbab117f8f99d94fedac6c4519b922e0/hedera-mirror-rosetta/container/entrypoint.sh#L31